internal/blueprint/customizations.go

```diff
@@ -88,6 +88,7 @@ type ServicesCustomization struct {
 
 type FilesystemCustomization struct {
 	Mountpoint string `json:"mountpoint,omitempty" toml:"mountpoint,omitempty"`
+	FSType     string `json:"fstype,omitempty" toml:"fstype,omitempty"`
```

Comment:
ftr, this is must be backward-compatible, which means it's optional and xfs is the default

Reply:
We discussed this a little further offline, I think the conclusion was a distro-specific default. For fedora it  would be ext4.